src/wombats/handlers/game.clj

```diff
@@ -123,7 +124,8 @@
 #_(s/def :game/players) ;; Add player specs
 
 (s/def :game/name string?)
-(s/def :game/max-players #(instance? Long %))
+(s/def :game/max-players #(and (instance? Long %)
```

Comment:
Interesting SO post for anyone curious why you need to check it's `Long`. [Link](http://stackoverflow.com/questions/9457537/why-does-int-10-produce-a-long-instance)